### PR TITLE
MS Service Bus Connector: Added field descriptions where none existed

### PR DESCRIPTION
**Pull request by dhirsch1**
This reference doc has many blank field descriptions. I added field descriptions where I could.

### CODE REVIEW

**Review by martinpaoloni — APPROVED** [review]

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 1]
```diff
@@ -60,71 +60,70 @@ Release Notes: xref:release-notes::connector/ms-service-bus-connector-release-no
...
+===== Shared Access Signature (Deprecated)
 
 ====== Parameters
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Skip connectivity test a| Boolean |  (Optional) Skips the connectivity test at connector startup (use this if your access policy is restricted). |  false | 
-| Max Connections a| Number |  Maximum number of connections to keep in pool to be reused by producer. If set to "-1" it will create a new connection everytime. |  -1 | 
+| Skip connectivity test a| Boolean |  (Optional) Skips the connectivity test at connector startup (use this if your access policy is restricted). |  false |
+| Max Connections a| Number |  Maximum number of connections to keep in pool to be reused by producer. If set to "-1" it will create a new connection everytime. |  -1 |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
| Max Connections a| Number |  Maximum number of connections to keep in pool for the producer to reuse.  If set to `-1`,  a new connection is created every time. |  -1 |
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 2]
```diff
@@ -60,71 +60,70 @@ Release Notes: xref:release-notes::connector/ms-service-bus-connector-release-no
...
+| Storage Account Name a| String |  (Optional) The account name of the Blob Storage to use to store large messages. |  |
+| Storage Access Key a| String |  (Optional) The account key of the Blob Storage to use to store large messages. |  |
+| Container Name a| String |  (Optional) The container name in the Blob Storage to use to store large messages. If this parameter is not provided a default container will be created |  |
+| Shared Access Signature a| String |  |  |
+| SAS tokens provider a| SharedAccessSignatureProvider |  Provider instance that will provide SAS tokens when required. |  |
+| SAS tokens list a| Array of String |  List of SAS tokens to use when required. |  |
+| Max redeliveries a| Number |  Controls when an incoming message is rejected based on the number of times it has been redelivered, the default value is (-1) disabled. A value of zero would indicate no message redeliveries are accepted, a value of five would allow a message to be redelivered five times, etc. |  -1 |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
| Max redeliveries a| Number |  Controls when an incoming message is rejected based on the number of times it has been redelivered. The default value is `-1` (disabled). A value of `0` (zero) means that no message redeliveries are accepted. A value of `5` means a message will be redelivered five times, and so on. |  -1 |
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 3]
```diff
@@ -60,71 +60,70 @@ Release Notes: xref:release-notes::connector/ms-service-bus-connector-release-no
...
 | Outcome a| Enumeration, one of:
 
 ** ACCEPTED
 ** REJECTED
 ** RELEASED
 ** MODIFIED_FAILED
-** MODIFIED_FAILED_UNDELIVERABLE |  Controls the outcome that is applied to a message that is being rejected due to it having exceeded the configured maxRedeliveries value. |  MODIFIED_FAILED_UNDELIVERABLE | 
-| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy |  | 
+** MODIFIED_FAILED_UNDELIVERABLE |  Controls the outcome that is applied to a message that is being rejected due to it having exceeded the configured maxRedeliveries value. |  MODIFIED_FAILED_UNDELIVERABLE |
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy |  |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. | `true` |
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 4]
```diff
@@ -60,71 +60,70 @@ Release Notes: xref:release-notes::connector/ms-service-bus-connector-release-no
...
 
 
 ====== Parameters
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Skip connectivity test a| Boolean |  (Optional) Skips the connectivity test at connector startup (use this if your access policy is restricted). |  false | 
-| Max Connections a| Number |  Maximum number of connections to keep in pool to be reused by producer. If set to "-1" it will create a new connection everytime. |  -1 | 
+| Skip connectivity test a| Boolean |  (Optional) Skips the connectivity test at connector startup (use this if your access policy is restricted). |  false |
+| Max Connections a| Number |  Maximum number of connections to keep in pool to be reused by producer. If set to "-1" it will create a new connection everytime. |  -1 |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
| Maximum number of connections to keep in pool for the producer to reuse.  If set to `-1`,  a new connection is created every time. |  -1 |
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 5]
```diff
@@ -60,71 +60,70 @@ Release Notes: xref:release-notes::connector/ms-service-bus-connector-release-no
...
 | Caching Strategy a| One of:
 
 * no-caching
-* <<default-caching>> |  The strategy to use for caching of Sessions and Connections |  | 
+* <<default-caching>> |  The strategy to use for caching of Sessions and Connections |  |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
* <<default-caching>> |  The strategy to use for caching Sessions and Connections |  |
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 6]
```diff
@@ -60,71 +60,70 @@ Release Notes: xref:release-notes::connector/ms-service-bus-connector-release-no
...
 | Service Namespace a| String |  |  | x
-| Storage Account Name a| String |  (Optional) The account name of the Blob Storage to use to store large messages. |  | 
-| Storage Access Key a| String |  (Optional) The account key of the Blob Storage to use to store large messages. |  | 
-| Container Name a| String |  (Optional) The container name in the Blob Storage to use to store large messages. If this parameter is not provided a default container will be created |  | 
+| Storage Account Name a| String | (Optional) The account name of the Blob Storage to use to store large messages |  |
+| Storage Access Key a| String | (Optional) The account key of the Blob Storage to use to store large messages. |  |
+| Container Name a| String |  (Optional) The container name in the Blob Storage to use to store large messages. If this parameter is not provided a default container will be created |  |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
| Container Name a| String |  (Optional) The container name in the Blob Storage to use to store large messages. If this parameter is not provided a default container is created. |  |
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 7]
```diff
@@ -60,71 +60,70 @@ Release Notes: xref:release-notes::connector/ms-service-bus-connector-release-no
...
-| Container Name a| String |  (Optional) The container name in the Blob Storage to use to store large messages. If this parameter is not provided a default container will be created |  | 
-| Shared Access Signature a| String |  |  | 
-| SAS tokens provider a| SharedAccessSignatureProvider |  Provider instance that will provide SAS tokens when required. |  | 
-| SAS tokens list a| Array of String |  List of SAS tokens to use when required. |  | 
-| Max redeliveries a| Number |  Controls when an incoming message is rejected based on the number of times it has been redelivered, the default value is (-1) disabled. A value of zero would indicate no message redeliveries are accepted, a value of five would allow a message to be redelivered five times, etc. |  -1 | 
+| Storage Account Name a| String |  (Optional) The account name of the Blob Storage to use to store large messages. |  |
+| Storage Access Key a| String |  (Optional) The account key of the Blob Storage to use to store large messages. |  |
+| Container Name a| String |  (Optional) The container name in the Blob Storage to use to store large messages. If this parameter is not provided a default container will be created |  |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
| Container Name a| String |  (Optional) The container name in the Blob Storage to use to store large messages. If this parameter is not provided a default container is created. |  |
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 8]
```diff
@@ -60,71 +60,70 @@ Release Notes: xref:release-notes::connector/ms-service-bus-connector-release-no
...
+| Storage Account Name a| String | (Optional) The account name of the Blob Storage to use to store large messages |  |
+| Storage Access Key a| String | (Optional) The account key of the Blob Storage to use to store large messages. |  |
+| Container Name a| String |  (Optional) The container name in the Blob Storage to use to store large messages. If this parameter is not provided a default container will be created |  |
 | Shared Access Key Name a| String |  |  | x
 | Shared Access Key a| String |  |  | x
-| Max redeliveries a| Number |  Controls when an incoming message is rejected based on the number of times it has been redelivered, the default value is (-1) disabled. A value of zero would indicate no message redeliveries are accepted, a value of five would allow a message to be redelivered five times, etc. |  -1 | 
+| Max redeliveries a| Number |  Controls when an incoming message is rejected based on the number of times it has been redelivered, the default value is (-1) disabled. A value of zero would indicate no message redeliveries are accepted, a value of five would allow a message to be redelivered five times, etc. |  -1 |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
| Max redeliveries a| Number |  Controls when an incoming message is rejected based on the number of times it has been redelivered. The default value is `-1` (disabled). A value of `0` (zero) means that no message redeliveries are accepted. A value of `5` means a message will be redelivered five times, and so on. |  -1 |
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 9]
```diff
@@ -60,71 +60,70 @@ Release Notes: xref:release-notes::connector/ms-service-bus-connector-release-no
...
 | Outcome a| Enumeration, one of:
 
 ** ACCEPTED
 ** REJECTED
 ** RELEASED
 ** MODIFIED_FAILED
-** MODIFIED_FAILED_UNDELIVERABLE |  Controls the outcome that is applied to a message that is being rejected due to it having exceeded the configured maxRedeliveries value. |  MODIFIED_FAILED_UNDELIVERABLE | 
-| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy |  | 
+** MODIFIED_FAILED_UNDELIVERABLE |  Controls the outcome that is applied to a message that is being rejected due to it having exceeded the configured maxRedeliveries value |  MODIFIED_FAILED_UNDELIVERABLE |
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy |  |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 10]
```diff
@@ -134,60 +133,60 @@ Release Notes: xref:release-notes::connector/ms-service-bus-connector-release-no
 [%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Skip connectivity test a| Boolean |  (Optional) Skips the connectivity test at connector startup (use this if your access policy is restricted). |  false | 
-| Max Connections a| Number |  Maximum number of connections to keep in pool to be reused by producer. If set to "-1" it will create a new connection everytime. |  -1 | 
+| Skip connectivity test a| Boolean |  (Optional) Skips the connectivity test at connector startup (use this if your access policy is restricted). |  false |
+| Max Connections a| Number |  Maximum number of connections to keep in pool to be reused by producer. If set to "-1" it will create a new connection everytime. |  -1 |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]: For Max Connections, same as previous suggestion

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 11]
```diff
@@ -134,60 +133,60 @@ Release Notes: xref:release-notes::connector/ms-service-bus-connector-release-no
...
 | Fully Qualified Domain Name a| String |  |  | x
-| Port a| String |  |  9355 | 
-| Disable SSL certificate validation a| Boolean |  |  false | 
-| Idle timeout a| Number |  The idle timeout in milliseconds after which the connection will be failed if the peer sends no AMQP frames. |  60000 | 
-| Max redeliveries a| Number |  Controls when an incoming message is rejected based on the number of times it has been redelivered, the default value is (-1) disabled. A value of zero would indicate no message redeliveries are accepted, a value of five would allow a message to be redelivered five times, etc. |  -1 | 
+| Port a| String |  |  9355 |
+| Disable SSL certificate validation a| Boolean |  |  false |
+| Idle timeout a| Number |  The idle timeout in milliseconds after which the connection will be failed if the peer sends no AMQP frames. |  60000 |
+| Max redeliveries a| Number |  Controls when an incoming message is rejected based on the number of times it has been redelivered, the default value is (-1) disabled. A value of zero would indicate no message redeliveries are accepted, a value of five would allow a message to be redelivered five times, etc. |  -1 |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]: Same as previous suggestion

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 12]
```diff
@@ -134,60 +133,60 @@ Release Notes: xref:release-notes::connector/ms-service-bus-connector-release-no
...
 | Outcome a| Enumeration, one of:
 
 ** ACCEPTED
 ** REJECTED
 ** RELEASED
 ** MODIFIED_FAILED
-** MODIFIED_FAILED_UNDELIVERABLE |  Controls the outcome that is applied to a message that is being rejected due to it having exceeded the configured maxRedeliveries value. |  MODIFIED_FAILED_UNDELIVERABLE | 
-| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy |  | 
+** MODIFIED_FAILED_UNDELIVERABLE |  Controls the outcome that is applied to a message that is being rejected due to it having exceeded the configured maxRedeliveries value. |  MODIFIED_FAILED_UNDELIVERABLE |
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy |  |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 13]
```diff
@@ -353,25 +352,25 @@ Sends a message to a Queue
 
 ** ALWAYS_JOIN
 ** JOIN_IF_POSSIBLE
-** NOT_SUPPORTED |  type of transactional action. |  NOT_SUPPORTED | 
+** NOT_SUPPORTED |  type of transactional action. |  NOT_SUPPORTED |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
** NOT_SUPPORTED |  Type of transactional action |  NOT_SUPPORTED |
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 14]
```diff
@@ -1049,19 +1043,19 @@ Sends a message to a Topic
...
-| Time To Live a| Number |  Defines the default time the message is in the broker before it expires and is discarded. |  | 
+| Body a| Any |  The body of the Message |  `#[payload]` |
+| JMS Type a| String |  The JMSType identifier header of the Message |  |
+| Correlation ID a| String |  The JMSCorrelationID header of the Message |  |
+| Send Content Type a| Boolean |  Whether or not the body content type should be sent as a property |  true |
+| Content Type a| String |  The content type of the message's body |  |
+| Send Encoding a| Boolean |  Whether or not the body outboundEncoding should be sent as a Message property |  true |
+| Encoding a| String |  The encoding of the message's body |  |
+| Reply To a| <<ServiceBusJmsDestination>> |  The destination where a reply to this Message should be sent |  |
+| User Properties a| Object |  The custom user properties that should be set to this Message |  |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
| User Properties a| Object |  The custom user properties that should be set for this Message |  |
```
??
This and the next couple of fields say "to this message" but I think it should be for?

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 15]
```diff
@@ -1197,29 +1191,29 @@ Retrieves all existing topics from the service bus instance
 ** AUTO
 ** MANUAL
 ** DUPS_OK |  The Session ACK mode to use when consuming a message |  |
-| Number Of Consumers a| Number |  The number of concurrent consumers to use to receive JMS Messages |  1 | 
-| Primary Node Only a| Boolean |  Whether this source should only be executed on the primary node when running in Cluster |  | 
-| Redelivery Policy a| <<RedeliveryPolicy>> |  Defines a policy for processing the redelivery of the same message |  | 
+| Number Of Consumers a| Number |  The number of concurrent consumers to use to receive JMS Messages |  1 |
+| Primary Node Only a| Boolean |  Whether this source should only be executed on the primary node when running in Cluster |  |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
| Primary Node Only a| Boolean |  Whether this source should be executed only on the primary node when running in a cluster |  |
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 16]
```diff
@@ -1197,29 +1191,29 @@ Retrieves all existing topics from the service bus instance
...
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  |
+| Body a| Any |  The body of the Message |  `#[payload]` |
+| JMS Type a| String |  The JMSType identifier header of the Message |  |
+| Correlation ID a| String |  The JMSCorrelationID header of the Message |  |
+| Send Content Type a| Boolean |  Whether or not the body content type should be sent as a property |  true |
+| Content Type a| String |  The content type of the message's body |  |
+| Send Encoding a| Boolean |  Whether or not the body outboundEncoding should be sent as a Message property |  true |
+| Encoding a| String |  The encoding of the message's body |  |
+| Reply To a| <<ServiceBusJmsDestination>> |  The destination where a reply to this Message should be sent |  |
+| User Properties a| Object |  The custom user properties that should be set to this Message |  |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
| User Properties a| Object |  The custom user properties to set for this message |  |
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 17]
```diff
@@ -1197,29 +1191,29 @@ Retrieves all existing topics from the service bus instance
...
+| JMSX Properties a| <<JmsxProperties>> |  The JMSX properties that should be set to this Message |  |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
| JMSX Properties a| <<JmsxProperties>> |  The JMSX properties to set for this message |  |
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 18]
```diff
@@ -1228,10 +1222,10 @@ Retrieves all existing topics from the service bus instance
...
 ** MINUTES
 ** HOURS
-** DAYS |  Time unit to use in the timeToLive configurations |  | 
-| Disable Message Id a| Boolean |  If true, the Message is flagged to avoid generating its MessageID |  | 
-| Disable Message Timestamp a| Boolean |  If true, the Message is flagged to avoid generating its sent Timestamp |  | 
-| Delivery Delay a| Number |  Only used by JMS 2.0. Sets the delivery delay to be applied to postpone the Message delivery |  | 
+** DAYS |  Time unit to use in the timeToLive configurations |  |
+| Disable Message Id a| Boolean |  If true, the Message is flagged to avoid generating its MessageID |  |
+| Disable Message Timestamp a| Boolean |  If true, the Message is flagged to avoid generating its sent Timestamp |  |
+| Delivery Delay a| Number |  Only used by JMS 2.0. Sets the delivery delay to be applied to postpone the Message delivery |  |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
| Delivery Delay a| Number |  Only used by JMS 2.0. Sets the delivery delay to apply to postpone the message delivery. |  |
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 19]
```diff
@@ -1370,11 +1364,11 @@ Retrieves all existing topics from the service bus instance
...
 |===
 | Field | Type | Description | Default Value | Required
-| Max Redelivery Count a| Number | The maximum number of times a message can be redelivered and processed unsuccessfully before triggering process-failed-message |  | 
-| Use Secure Hash a| Boolean | Whether to use a secure hash algorithm to identify a redelivered message. |  | 
-| Message Digest Algorithm a| String | The secure hashing algorithm to use. If not set, the default is SHA-256. |  | 
-| Id Expression a| String | Defines one or more expressions to use to determine when a message has been redelivered. This property may only be set if useSecureHash is false. |  | 
-| Object Store a| Object Store | The object store where the redelivery counter for each message is going to be stored. |  | 
+| Max Redelivery Count a| Number | The maximum number of times a message can be redelivered and processed unsuccessfully before triggering process-failed-message |  |
+| Use Secure Hash a| Boolean | Whether to use a secure hash algorithm to identify a redelivered message. |  |
+| Message Digest Algorithm a| String | The secure hashing algorithm to use. If not set, the default is SHA-256. |  |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
| Message Digest Algorithm a| String | The secure hashing algorithm to use.  | SHA-256 |
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc [hunk 20]
```diff
@@ -1370,11 +1364,11 @@ Retrieves all existing topics from the service bus instance
...
+| Id Expression a| String | Defines one or more expressions to use to determine when a message has been redelivered. This property may only be set if useSecureHash is false. |  |
```
Comment (valkyrie69) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector-reference.adoc]:
```suggestion
| Id Expression a| String | Defines one or more expressions to use to determine when a message has been redelivered. This property can be set only if useSecureHash is false. |  |
```

**Review by valkyrie69 — APPROVED** [review]
Approved with some suggestions. Thank you for improving the doc!

**Pull request merged**